.github/workflows/test.yml

```diff
@@ -18,47 +18,37 @@ jobs:
           - name: latest
             version: latest
         ruby_version: ["3.2.2"]
-    name: Rails tests (RubyGems ${{ matrix.rubygems.name }})
+        tests:
```

Comment:
it will be easier to manage the required checks if we add a new job that `needs` this test job (to act as a fan-in for test success)

Reply:
I'm not I do fully follow in here. Do you recommend to create new test job with `needs: [rails]` doing nothing in the end (it can just print "tests passing", I'm not sure empty job is supported) and use that one in checks on GH side for required checks setup?

Reply:
yup, exactly that!

Reply:
I'll try.

Reply:
@segiddins feel free to check. My plan is to add comment like this to explain.

```
# This is umbrella job for all tests needed to pass to make it simpler
# to maintain GitHub Actions status required checks since job names and
# amount change over the time and it is easier to maintian having just
# this umbrella check set as required.
```

Feel free to propose better wording if needed.